src/hash/hash.c

```diff
@@ -2173,6 +2178,34 @@ static bool setResultFromEvidence(
 	return EXCEPTION_OKAY;
 }
 
+static bool setResultFromDeviceID(
+	void* state,
+	EvidenceKeyValuePair* pair) {
+
+	if (StringCompare(pair->field, "51D_deviceId")) {
+		return true; // not a match, skip
```

Comment:
are we ok returning true from this method, does it mean "go on with other evidence?"

Reply:
Yes, `true` means 'continue iterating over evidence pairs':
- https://github.com/51Degrees/common-cxx/blob/main/evidence.c#L120